lib/libraries/default/message.js

```diff
@@ -2,20 +2,31 @@ var escapeHtml = require('../../utils/escape-html');
 
 module.exports = {
   render: function(text){
-    var outer = document.createElement('div'),
-        inner = document.createElement('div'),
-        msg = document.createElement('span'),
-        height = this.height() || 140;
+    var outer = document.createElement('div');
+    var inner = document.createElement('div');
+    var msg = document.createElement('span');
+    var height = this.height() || 140;
+
+    var titleContainer = document.createElement('div');
+    var notesContainer = document.createElement('div');
 
     outer.className = this.theme();
     inner.className = this.theme() + '-message';
     inner.style.height = height + 'px';
```

Comment:
We need to factor in `this.el().offsetHeight` here so that the title and notes elements' height (and borders, margins, padding, etc) will be accounted for in the final height. Here's how it's done for C3.js artifacts: https://github.com/keen/keen-dataviz.js/blob/d02dfb6fcbcf125bdfbde186409990e0f1df93c1/lib/libraries/index.js#L103